recipe/meta.yaml

```diff
@@ -63,26 +62,20 @@ outputs:
         - coveralls >=1.8.0
         - doxygen >=1.9.8
         - eups >=2.2.8
-        - flake8 ==7.1.0
-        - libblas * *openblas
-        - openblas
+        - flake8 ==7.1.1
         - make >=4.3
         - meson >=1.3.0
-        - nose >=1.3.7
         - pep8-naming >=0.13.3
         - pip >=23.3.2
-        - pkg-config >=0.29.2
         - psutil >=5.9.7
         - pytest >=8.2.0
         - pytest-cov >=4.1.0
         - pytest-doctestplus >=1.1.0
-        - pytest-openfiles >=0.5.0
         - pytest-runner >=6.0.0
         - pytest-session2file >=0.1.11
         - pytest-subtests >=0.11.0
         - pytest-vcr >=1.0.2
         - pytest-xdist >=3.5.0
-        - setuptools <=72.1.0
```

Comment:
I'd keep setuptools even if you remove the pin. Otherwise I am betting builds will start to break. People expect it to be there.